<launch>
	<node pkg="px4_command" type="collision_avoidance" name="collision_avoidance" output="screen">
                <rosparam command="load" file="$(find px4_command)/config/cross_door.yaml" />
	</node>
</launch>
